src/t8_geometry/t8_geometry_implementations/t8_geometry_occ.hxx

```diff
@@ -250,16 +251,73 @@ public:
    *  The edge has to lie on the face.
    * \param [in]  edge_index     The index of the occ edge.
    * \param [in]  face_index     The index of the occ face.
+   * \param [in]  num_face_nodes The number of the face nodes.
    * \param [in]  edge_param     The parameter on the edge.
+   * \param [in]  surface_param  The parameters of the surface nodes.
+   *                             When provided, there are additional checks for closed geometries.
+   *                             If there are no surface parameter
+   *                             to pass in to the function, you can pass NULL.
```

Comment:
This still does not explain, what exactly the function wants and does. Especially, the `num_face_nodes` argument is not explained enough. This function normally has nothing to do with a mesh and its nodes, it just converts parameters on geometries. If you want to have mesh nodes inserted, you have to explain the use case in more detail and especially focus on the difference of an element face and a topological face. There is no such thing as too much documentation.

Reply:
It should be mentioned, that `num_face_nodes` is also an optional argument and there should be a way to also pass `NULL` here as well. Also, optional arguments should always be put last in an argument list.